lib/components/ClinicUserEdit.js

```diff
@@ -19,11 +19,37 @@ var React = require('react');
 var _ = require('lodash');
 var sundial = require('sundial');
 var personUtils = require('../core/personUtils');
+import { reduxForm } from 'redux-form';
 
 var config = require('../config');
 
 var styles = require('../../styles/components/ClinicUserEdit.module.less');
 
+var zeroPad = function(value){
+  return _.padLeft(value, 2, '0');
+};
+
+var validateForm = function(values){
+  var errors = {};
+  if(!values.fullName){
+    errors.fullName = 'Name is required';
+  }
+  if(values.year && values.month && values.day){
+    if(!isValidDate(values.year + '-' + values.month + '-' + zeroPad(values.day))){
+      errors.year = 'Invalid date';
+    }
+  } else {
+    errors.year = 'Invalid date';
+  }
+  return errors;
+};
+
+var isValidDate = function(dateString){
```

Comment:
I slightly prefer non-anonymous functions (e.g., `function isValidDate(dateString) { ... }` or just adding the function name again in addition to assigning to the variable) because they are easier to debug in the console, but not a biggie. If you have strong feelings on a preference, would be interested to hear.